error.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <qiskit-ui-shell variant="hide-account" @on-click="onClick" />
+    <div class="error-page">
+      <div class="cds--grid">
+        <div class="cds--row">
+          <div class="cds--col-lg-4">
+            <h1 class="error-page__title">{{ props.error?.statusCode }}</h1>
+            <p>
+              {{ props.error?.message }}
+            </p>
+            <p class="error-page__message">
+              Or as we like to say,
+              <!-- eslint-disable vue/no-v-html -->
+              <span v-html="randomMessage" />
+              <!-- eslint-enable -->
+            </p>
+
+            <UiCta
+              :label="BackToHomeCTA.label"
+              :segment="BackToHomeCTA.segment"
+              :url="BackToHomeCTA.url"
+            />
+            <UiCta
+              class="error-page__ghost-btn"
+              :label="BackToDocsHome.label"
+              :segment="BackToDocsHome.segment"
+              :url="BackToDocsHome.url"
+              kind="ghost"
+            />
+          </div>
+          <div class="cds--col-lg-8">
+            <img
+              class="error-page__img"
+              :src="errorImgSrc"
+              alt="Playful illustration of a cat in a cardboard box"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+    <FooterComponent old-container theme="light" />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { GeneralLink } from "./constants/appLinks";
+
+const props = defineProps({
+  // eslint-disable-next-line vue/require-default-prop
+  error: Object,
+});
+
+const BackToHomeCTA: GeneralLink = {
+  url: "/",
+  label: "Back to Qiskit.org home",
+  segment: {
+    cta: "back-to-home",
+    location: "error-page",
+  },
+};
+
+const BackToDocsHome: GeneralLink = {
```

Comment:
We usually camel-case the variables:

```suggestion
const backToDocsCta: Link = {
```